tests/acceptance/project-settings-test.js

```diff
@@ -85,6 +85,35 @@ test("it allows editing of project's image for owners", function(assert) {
   });
 });
 
+test("it allows editing of project's categories for owners", function(assert) {
+  assert.expect(4);
+
+  server.create('category', { name: 'Technology' });
+  let { project, user } = server.create('project-user', { role: 'owner' });
+
+  authenticateSession(this.application, { user_id: user.id });
+
+  projectSettingsPage.visit({
+    organization: project.organization.slug,
+    project: project.slug
+  });
+
+  andThen(() => {
+    assert.equal(currentURL(), `${project.organization.slug}/${project.slug}/settings/profile`, 'The project settings profile page loaded');
+    assert.equal(projectSettingsPage.categoryCheckboxes.checkboxes(0).label.name, 'Technology', 'The checkbox renders');
+    projectSettingsPage.categoryCheckboxes.checkboxes(0).label.click();
+  });
+
+  andThen(() => {
+    assert.ok(projectSettingsPage.categoryCheckboxes.checkboxes(0).isChecked, 'The category was added.');
+    projectSettingsPage.categoryCheckboxes.checkboxes(0).label.click();
+  });
+
+  andThen(() => {
+    assert.notOk(projectSettingsPage.categoryCheckboxes.checkboxes(0).isChecked, 'The category was removed.');
+  });
+});
+
```

Comment:
I would maybe add some assertions on the data here, instead of just checking the UI looks good.